products/compute/puppet.yaml

```diff
@@ -569,6 +569,8 @@ files: !ruby/object:Provider::Config::Files
       templates/puppet/bolt~task_load_params.rb.erb
     lib/puppet/functions/gcompute_task_validate_param.rb:
       templates/puppet/bolt~task_validate_param.rb.erb
+    lib/puppet/functions/gcompute_external_resource.rb:
```

Comment:
Move this to `provider/puppet/common~compile.yaml` if we decide that this is not just a compute thing.

Reply:
Done.